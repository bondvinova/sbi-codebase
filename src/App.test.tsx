// import { render, screen } from '@testing-library/react';
import { MemoryRouter, Route, Routes } from 'react-router-dom';
import { create } from 'react-test-renderer';
import { describe, expect, it } from 'vitest';

import Login from '@/features/authentication/login';

describe('App', () => {
  it('renders correctly', () => {
    const renderer = create(
      <MemoryRouter initialEntries={['/login']}>
        <Routes>
          <Route path="/login" element={<Login />} />
        </Routes>
      </MemoryRouter>
    );

    expect(renderer.toJSON()).toMatchSnapshot();
  });
});
