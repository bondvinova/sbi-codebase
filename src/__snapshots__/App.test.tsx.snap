// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`App > renders correctly 1`] = `
<div>
  Login
</div>
`;
